pheme/templatetags/charts/__init__.py

```diff
@@ -66,19 +71,19 @@ def _build_legend(
     """
     legend_elements = ""
     x_pos = 0
+    y_pos = 0
+    text_x = font_size + 2
```

Comment:
```suggestion
    text_x_offset = font_size + 2
```

Reply:
It is the position of text element not an offset based on a text element. So I guess following _pos schema would be better?

Reply:
yeah pos is okay.
I just needed some time to figure out that this is moved in the x direction because there is the color rectangle in front of it ... :)

Reply:
true, I added a comment for the two cases.